Improved getting started page

AaronB233 opened the pull request:
Added a brief summary for each of the project purposes and the main steps for the project creation process.  Added cards for rural and urban projects with a brief description for each.  Lastly, added timeline information with a link to the main timeline page.

Blade-iii reviewed (changes requested):
Good work!
Nice explanation of the process and explains it very well

Some improvements
- Move the html code into info folder as it makes sense to put it there with the other info pages.
- Route change `/getting-started` for example within routes.py 
- Open the timeline in new tab this can be achieved by doing :
`<a href="{{ url_for('timeline') }}"  target="_blank" class="form-control btn btn-danger border border-black px-5">View Full Timeline</a>`

NikitaEdin reviewed (changes requested):
The page looks great, but there's no route defined for this page or any possible way for the user to navigate to it.

Could you add a route and ensure there's a way for users to access the page?
It could be a button on the dashboard, the navbar, or the homepage.

AaronB233 commented:
I have implemented the suggested improvements, the 'get started' button on the home page now links to the getting-started page.  I also added a button at the bottom of the 'project creation process' card that links to the dashboard.

NikitaEdin reviewed (approved):
Well done, the page looks great and well-structured!

> [!TIP]
>  For buttons, use `btn-primary` class tag to keep it consistent.

Blade-iii reviewed (approved):
Good work!

Thank you for implementing the changes!
